ink/crates/phat_rollup_anchor_ink/src/traits/meta_transaction.rs

```diff
@@ -179,3 +139,16 @@ pub trait MetaTransaction:
 pub trait EventBroadcaster {
     fn emit_event_meta_tx_decoded(&self);
 }
+
+/// Hashing function for bytes
+fn hash_blake2b256(input: &[u8]) -> [u8; 32] {
+    use ink::env::hash;
+    let mut output = <hash::Blake2x256 as hash::HashOutput>::Type::default();
+    ink::env::hash_bytes::<hash::Blake2x256>(input, &mut output);
+    output
+}
+
+/// Converts a compressed public key to SS58 format
+fn pub_key_to_ss58(pub_key: &[u8; 33]) -> AccountId {
```

Comment:
Nit: `AccountId` is just `[u8; 32]` under the hood. SS58 only kicks in when it's displayed on the frontend. So I suggest to name it `get_ecdsa_account_id()`

Reply:
I renamed the function